tests/mir-opt/const_prop/mutable_variable.main.ConstProp.diff

```diff
@@ -17,8 +17,7 @@
           _1 = const 42_i32;               // scope 0 at $DIR/mutable_variable.rs:+1:17: +1:19
           _1 = const 99_i32;               // scope 1 at $DIR/mutable_variable.rs:+2:5: +2:11
           StorageLive(_2);                 // scope 1 at $DIR/mutable_variable.rs:+3:9: +3:10
--         _2 = _1;                         // scope 1 at $DIR/mutable_variable.rs:+3:13: +3:14
-+         _2 = const 99_i32;               // scope 1 at $DIR/mutable_variable.rs:+3:13: +3:14
+          _2 = _1;                         // scope 1 at $DIR/mutable_variable.rs:+3:13: +3:14
```

Comment:
what happened here?

Reply:
`_1` is assigned twice, and this pass is the cheap version limited to SSA locals.